Lib/test/test_http_cookies.py

```diff
@@ -215,11 +215,18 @@ def test_pickle(self):
                 self.assertEqual(C1.output(), expected_output)
 
     def test_illegal_chars(self):
-        rawdata = "a=b; c,d=e"
+        rawdata = "a=b; c@d=e"
```

Comment:
I think comma in cookie's key were explicitly prohibited as per this test which fixed a regression that allowed it. Reference : https://bugs.python.org/issue26302

Reply:
They are prohibited as key but this patch allow them as separator so the string `"a=b; c,d=e"` would be equivalent as `"a=b; c;d=e"` which  will be ignored by `SimpleCookie.load`

Reply:
Ah okay, I was slightly confused since there was change in `_LegalKeyChars` and this test was changed. Thanks for the clarification. I don't know if it's redundant to add the previous test to `test_comma_separator` to demonstrate the fact that "a=b; c,d=e" is equivalent to "a=b; c;d=e" and ignored by `SimpleCookie.load`. I will let others to decide on the same.

Reply:
Thanks, an explicit test is better anyway than supposing the reader will have knowledge of the b.p.o issue and will understand. I added one.